File: app/src/main/java/com/antyzero/smoksmog/ui/screen/start/PageSave.java
package com.antyzero.smoksmog.ui.screen.start;

import android.content.Context;
import android.content.SharedPreferences;

public class PageSave {

    private static final String TAG = PageSave.class.getSimpleName();

    private static final String KEY_ORDER = "keyOrder";

    private final SharedPreferences preferences;

    public PageSave(Context context) {
        preferences = context.getSharedPreferences(TAG, Context.MODE_PRIVATE);
    }

    public void savePage(int pageOrder) {
        preferences.edit().putInt(KEY_ORDER, pageOrder).apply();
    }

    public int restorePage() {
        return preferences.getInt(KEY_ORDER, 0);
    }
}

File: app/src/main/kotlin/com/antyzero/smoksmog/ui/screen/start/PageSave.kt
package com.antyzero.smoksmog.ui.screen.start

import android.content.Context
import android.content.SharedPreferences

class PageSave(context: Context) {

    private val preferences: SharedPreferences

    init {
        preferences = context.getSharedPreferences(TAG, Context.MODE_PRIVATE)
    }

    fun savePage(pageOrder: Int) {
        preferences.edit().putInt(KEY_ORDER, pageOrder).apply()
    }

    fun restorePage(): Int {
        return preferences.getInt(KEY_ORDER, 0)
    }

    companion object {

        private val TAG = PageSave::class.java.simpleName

        private val KEY_ORDER = "keyOrder"
    }
}
